partiql-spi/src/main/kotlin/org/partiql/spi/utils/FunctionUtils.kt

```diff
@@ -136,7 +139,8 @@ internal object FunctionUtils {
     }
 
     internal fun logicalNot(a: Datum): Datum {
-        if (isUnknown(a)) return Datum.nullValue(PType.bool())
+        if (a.isMissing) return Datum.missing(PType.bool())
+        if (a.isNull) return Datum.nullValue(PType.bool())
```

Comment:
Ah sorry for the confusion here. The original conformance test I had added was incorrect. `NOT MISSING` should return `NULL`. I'll fix the test.

Reply:
Test fixed in https://github.com/partiql/partiql-tests/pull/153.